### ISSUE TITLE
Relationship to Frictionless Data

### ISSUE THREAD

**Issue by nealmcb** [openelections/specs#3]
The repo README.md points to [Open Data Protocols](http://www.dataprotocols.org/en/latest/) which now redirects to http://specs.frictionlessdata.io/
Are the current openelections specs related to the frictionlessdata specs?

**Comment by dwillis** [comment]
No, they aren't. The link was there for reference, but we haven't had any discussion or action about the ODP work.
